src/lib_c/x86_64-windows-msvc/c/ntsecapi.cr

```diff
@@ -0,0 +1,6 @@
+require "./winnt"
+
+@[Link("advapi32")]
+lib LibC
+  fun RtlGenRandom = SystemFunction036(random_buffer : Void*, random_buffer_length : ULong) : BOOLEAN
```

Comment:
Why the alias?

Reply:
Nevermind, I guess it's a #define

Reply:
RtlGenRandom is what it's called in the windows documentation. The point of keeping the windows names is to be able to find the documentation easilly, so I think this is correct. It's much easier to read this way anyway.

Reply:
Have a look to libsodium sysrandom. The function is known as RtlGenRandom, but the  symbol is `SystemFunction036` and kind of private API, but everybody uses it (libsodium, go, etc) because it avoids bundling the whole crypto API, which is apparently quite big.

An alias, here, is expected, yes :-)